apps/payments/next/app/[locale]/subscriptions/manage/page.tsx

```diff
@@ -67,644 +72,807 @@ export default async function Manage({
+                                `Expires ${expirationDate}`
+                              )}
+                            </p>
+                          )}
+                        </div>
+                        <Link
+                          className="bg-grey-10 border border-grey-200 box-border font-bold font-header inline-block rounded text-center w-full py-2 px-5 tablet:w-auto"
+                          href={`${config.paymentsNextHostedUrl}/${locale}/subscriptions/payments/stripe`}
+                          aria-label={l10n.getString(
+                            'subscription-management-button-manage-payment-method-aria',
+                            'Manage payment method'
+                          )}
+                        >
+                          {l10n.getString(
+                            'subscription-management-button-manage-payment-method',
+                            'Manage'
+                          )}
+                        </Link>
+                      </div>
                     )}
-                  >
-                    <span>
-                      {l10n.getString(
-                        'subscription-management-button-manage-payment-method',
-                        'Manage'
-                      )}
-                    </span>
-                  </LinkExternal>
-                ) : (
-                  <LinkExternal
-                    className={CSS_SECONDARY_LINK}
-                    href={`${config.csp.paypalApi}/myaccount/autopay/connect/${billingAgreementId}`}
-                    aria-label={l10n.getString(
-                      'subscription-management-button-change-payment-method-aria',
-                      'Change payment method'
+
+                    {type === 'link' && (
+                      <div className="w-full flex items-center justify-between">
+                        <Image
+                          src={getCardIcon('link', l10n).img}
+                          alt={l10n.getString(
+                            'link-logo-alt-text',
+                            'Link logo'
+                          )}
+                          width={72}
+                          height={24}
+                        />
+                        <Link
+                          className="bg-grey-10 border border-grey-200 box-border font-bold font-header inline-block rounded text-center py-2 px-5 w-auto"
+                          href={`${config.paymentsNextHostedUrl}/${locale}/subscriptions/payments/stripe`}
+                          aria-label={l10n.getString(
+                            'subscription-management-button-manage-payment-method-aria',
+                            'Manage payment method'
+                          )}
+                        >
+                          {l10n.getString(
+                            'subscription-management-button-manage-payment-method',
+                            'Manage'
+                          )}
+                        </Link>
```

Comment:
suggestion (non-blocking): do not duplicate <Link> for Manage payment method 

The Mange payment method Link/button appears to be the same for card, wallets and Link payment methods. Instead of duplicating the Link, could just the Image change depending on what type of payment method it is?

Reply:
Per discussion, I felt it would be easier to refactor the Payment method section when working on the ticket that adds in the warning messages, as this refresh ticket already moves around a lot of things.